README.md

```diff
@@ -29,12 +29,8 @@ Select your `lambda.zip` as the "Function code" and make the handler "index.hand
 
 ![Function code](https://raw.githubusercontent.com/lambci/node-custom-lambda/master/img/function_code.png "Function code setup screenshot")
 
-Then click on Layers and choose "Add a layer", and "Provide a layer version
-ARN" and enter the following ARN:
 
-```
-arn:aws:lambda:us-east-1:553035198032:layer:nodejs10:1
-```
+Then click [arn:aws:lambda:us-east-1:553035198032:layer:nodejs10:1](https://console.aws.amazon.com/lambda/home?region=us-east-1#/connect/layer?layer=arn:aws:lambda:us-east-1:553035198032:layer:nodejs10:1) and pick your function from the "Function name" auto-suggest.
```

Comment:
How about:
```suggestion
Then enter `arn:aws:lambda:us-east-1:553035198032:layer:nodejs10:1` (or [use this link](https://console.aws.amazon.com/lambda/home?region=us-east-1#/connect/layer?layer=arn:aws:lambda:us-east-1:553035198032:layer:nodejs10:1)) and pick your function from the "Function name" auto-suggest.
```

Reply:
Or:

"Then add a layer with the following ARN: ... (or [use this link])"

Reply:
I guess you want to keep both ways to enter the layer version ARN. To use the previous one, we need to click the "Layers" card first, so the whole paragraph would be:

Then click on Layers and choose "Add a layer", and "Provide a layer version ARN" and enter the following ARN:

```
arn:aws:lambda:us-east-1:553035198032:layer:nodejs10:1
```

Or [use this link](https://console.aws.amazon.com/lambda/home?region=us-east-1#/connect/layer?layer=arn:aws:lambda:us-east-1:553035198032:layer:nodejs10:1) and pick your function from the "Function name" auto-suggest.

Reply:
Yeah, I would like to keep both ways, just because ppl aren't necessarily going to trust a link, or at least know what it does (plus it's nice to be explicit) 👍